src/components/Navbar/Navbar.js

```diff
@@ -1,9 +1,116 @@
-import React from 'react'
+import { AppBar, Toolbar, Typography } from '@mui/material'
+import {React, useState, Fragment} from 'react'
 import styles from '../Navbar/navbar.module.scss'
+import Box from '@mui/material/Box';
+import Avatar from '@mui/material/Avatar';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import Tooltip from '@mui/material/Tooltip';
+import InputSearch from './InputSearch/InputSearch';
+
 const Navbar = () => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  
   return (
     <>
-  
+    <AppBar position='fixed' color='primary' sx={{display:'flex'}}>
+      <Toolbar sx={{justifyContent:'space-between'}} >
+        <Typography variant='h6'>
+          COURSE-ON
+        </Typography>
+        <InputSearch/>
+        <Fragment>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
+        <Typography sx={{ minWidth: 100 }}>About</Typography>
+        <Tooltip title="Account settings">
+          <IconButton
+            onClick={handleClick}
+            size="small"
+            sx={{ ml: 2 }}
+            aria-controls={open ? 'account-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
```

Comment:
```suggestion
            aria-expanded={open && 'true'}
```

De todos modos, sera undefined y el por defecto es `false`.

https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaHasPopup